releasenotes/notes/remove-deprecated-gate-args-b29ba17badcb77ca.yaml

```
@@ -0,0 +1,7 @@
+---
+deprecations:
```

This should be an upgrade release note because it's something a user will potentially need to change when upgrading to the next version

```suggestion
upgrade:
```